service/src/main/java/bio/terra/cbas/dao/RunDao.java

```diff
@@ -20,4 +28,33 @@ public int createRun(Run run) {
             + " values (:id, :engineId, :runSetId, :entityId, :submissionTimestamp, :status)",
         new BeanPropertySqlParameterSource(run));
   }
+
+  public List<Run> retrieve() {
+    String sql =
+        "SELECT * FROM run INNER JOIN run_set ON run.run_set_id = run_set.id"
+            + " INNER JOIN method ON run_set.method_id = method.id";
```

Comment:
I am a bit surprised we are not using some kind of abstraction layer, but selecting such would probably be beyond the scope of this story

Reply:
This is also what happens in "the other Terra Tech Stack projects we copied from", FWIW